docs/how-to/debug/use-the-library-linter.rst

```diff
@@ -3,4 +3,40 @@
 Use the library linter
 ======================
 
-.. copy "Fix flagged issues" section
+The following guidelines describe how to address issues flagged by the library linter.
+
+To resolve a missing dependency, add the missing package to the part's
+``stage-packages`` key.
+
+To resolve an unused library:
+
+.. list-table::
+   :header-rows: 1
+
```

Comment:
Please add a `:widths:` value (probably `1 3`) that makes all the words in the first column's cells appear on the same line.